tasks/main.yml

```diff
@@ -177,6 +177,13 @@
     group: "{{ tomcat_user_group }}"
     mode: 0640
 
+- name: Remove ROOT Webapp Context
+  file:
+    state: absent
+    path: "{{ tomcat_instance_path }}/webapps/ROOT/"
+  when:
+    - not tomcat_root_context
```

Comment:
Also, as long as we are looking to remove the default webapps, perhaps rather than just the root context, but perhaps a list webapps to remove on first install. With us defaulting to leaving them but when true removing all from the array, the default probably leaving manager in place but removing the rest. 

Ideas?